scripts/create_bridges.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Service to create bridges
+Before=libvirtd.service
+After=network-online.target 
+After=network.target
+
+[Service]
+WorkingDirectory = /etc/
+ExecStart=/usr/bin/create_bridges
+Type=simple
```

Comment:
@henn I am not sure about the race condition with `libvirt`. I think the case using `oneshot` is that user will have to do `systemctl start create-bridge.service` every time changes have been made to the `hil.cfg`, since the systemd service will run only once and exit. 
If it is the case, then it makes no difference between using this systemctl script and use bash script at all.